### ISSUE TITLE
Show IRI when hovering mouse over node in tree.

### ISSUE THREAD

**Issue by henrietteharmse** [EBISPOT/ols4#452]
Compare with OLS 3 for example by hovering over any node in the tree.

![Screenshot from 2023-08-03 11-35-27](https://github.com/EBISPOT/ols4/assets/32327979/05c31615-2c3a-473e-8222-514ee4e67ec0)

**Comment by rombaum** [comment]
Hi @serjoshua ,

I had a discussion about this missing information in the tree with @julian-schneider at the end of last week. 

Before starting a new issue I searched for an already existing issue and found this one. So nevertheless I was checking if this issue still exists at your ols4 instance and I couldn't find the IRI as a tooltip yet (e.g. [diabetes mellitus(EFO:0000400)](https://www.ebi.ac.uk/ols4/ontologies/efo/classes/http%253A%252F%252Fwww.ebi.ac.uk%252Fefo%252FEFO_0000400)). 

So is this really fixed and the website was not updated yet or is there maybe still an issue there?

**Comment by serjoshua** [comment]
Hello @rombaum, we already did changes to fix this. You could refer to https://github.com/EBISPOT/ols4/pull/458. The problem is we encountered an issue last Friday that forced us to re-deploy a previous build, which did not include the fix to this issue. We have no specific timeline yet for the next code deployment to production so I will re-open this issue for monitoring purposes but if you have questions regarding the fix, let us know. Thanks.
